### DISCUSSION TITLE
Using Template literals/strings to avoid repetition

### DISCUSSION THREAD

**Discussion by sebgok** [jspsych/jsPsych#2925] [Q&A]
Hello, 

Across my trials, I use a lot of the same strings, so the code is too repetitive. I turned the strings into variables with the hope of using them as template literals/strings, but it didn't work. I'm wondering if there is any other solution to this?  What I want to achieve is something like this: 

```
let welcome = "hello"

var trial = {
type: jsPsychInstructions, 
pages: [`${welcome}`]
},

var trial2 = {
type: jsPsychInstructions, 
pages: [`${welcome}`]
},

var trial3 = {
type: jsPsychInstructions, 
pages: [`${welcome}`]
},
```

**Comment by jodeleeuw** [comment]
Hi @sebgok 

This should work. I'm guessing the problem is specific to something else in the code. Do you want to share more of the actual code?
